src/EbmlCrc32.cpp

```diff
@@ -204,7 +208,11 @@ bool EbmlCrc32::CheckElementCRC32(EbmlElement &ElementToCRC)
   MemIOCallback memoryBuffer;
   ElementToCRC.Render(memoryBuffer);
 
-  return CheckCRC(m_crc_final, memoryBuffer.GetDataBuffer(), memoryBuffer.GetDataBufferSize());
+  uint64 memSize = memoryBuffer.GetDataBufferSize();
+  if (memSize > UINT32_MAX)
```

Comment:
Why use all these ugly C-style preprocessor macros? Why not use the type system as it's supposed to be used, as in `std::numeric_limits<uint32>::max()`?

Reply:
1. because I don't know about it
2. ugly is a matter of taste

I can use that. BTW I'd also like to get rid (keep for backward compatibility) our custom versions of standard types.

Reply:
> BTW I'd also like to get rid (keep for backward compatibility) our custom versions of standard types.

As in: keep the definitions, but change all of our code to use the `…_t` variants from `stdint`? Definitely in favor!